database/schemaregistry/10_data/04_tables/01_event_type.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS zsr_data.event_type (
+  et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
+  et_event_type_object jsonb NOT NULL
```

Comment:
probably would not be bad having a further constraint checking that the name field (within the event type object) matches et_name.